docs/topic-guides/aws.txt

```diff
@@ -8,8 +8,8 @@ on AWS.
 Overview
 ~~~~~~~~
 
-A `master` node (a single, non-GPU instance) manages the cluster, provisioning
-and terminating `agent` nodes dynamically as new workloads are started by users.
+A master node (a single, non-GPU instance) manages the cluster, provisioning
```

Comment:
non-blocking: Not for this review, but it could make sense to make Master and Agent proper nouns within the Determined context.